en/contributing/build.md

```diff
@@ -254,23 +254,32 @@ You can also build the image yourself using the [Dockerfile](https://github.com/
    ```
 
 
-## Install Artifacts
+## Install DroneCore Headers and Static Library
 
-The build artifacts (headers and static library file) can be installed locally into the folder `./install/` using:
+In order to *use* DroneCore your C++ applications need to be able locate the DroneCore libraries and header files (see [Building C++ Apps](../guide/toolchain.md)). You can either install these to standard system locations or locally within the DroneCore directory.
 
-```
-make default install
-```
+> **Note** When using the library **libdronecore.a**, you need to link to a thread library such as *pthread* on a POSIX system (pthread is not included in the static library because it is included in glibc).
 
-Or, to install the files system-wide, using:
 
-```
-INSTALL_PREFIX=/usr/local make default install
+### System-wide Install
+
+Installing DroneCore system-wide means that the files you need to build your apps are always in the same place, and your build definition files can be simpler because they do not need to consider relative locations.
+
+> **Warning** System-wide install is not (yet) supported on Windows. If you're a Windows guru, we'd [love your help](../README.md#getting-help) to set this up.
```

Comment:
@maetugr you do Windows, don't you?

Reply:
Yes but I almost always tried to work with MinGW/msys and GCC not the official Visual Studio way.
But I see that by the time I try to get my notifications under control you already helped yourself.

Reply:
Well, we haven't solved native windows build yet. Not sure if MinGW is "good enough" solution. @julianoes ?

Reply:
Last time I tried MinGW I could not stand it.

Reply:
My take on this is that I'm very impressed by what @MaEtUgR has done with MinGW but I see it as a bit "non standard". The learning curve is a bit high if you want to configure or change anything. 
My preference would be to use VisualStudio (which I see as the standard). Either way, we need to make this turnkey.